cpp/src/c_api/graph_mg.cpp

```diff
@@ -153,6 +170,11 @@ struct create_graph_functor : public cugraph::c_api::abstract_functor {
           std::move(edgelist_edge_ids),
           std::move(edgelist_edge_types));
 
+      if (vertex_list) {
+        vertex_list = cugraph::detail::shuffle_ext_vertices_to_local_gpu_by_vertex_partitioning(
+          handle_, std::move(*vertex_list));
+      }
```

Comment:
If we are assuming that edges are pre-shuffled, shouldn't we also assume that vertices are also pre-shuffled? I guess we'd better be consistent.

Reply:
We are not assuming things are reshuffled.

Reply:
Oh I see, I missed the cugraph::detail::shuffle_ext_vertex_pairs_with_values_to_local_gpu_by_edge_partitioning() call.